nengo_bones/scripts/check_deploy.py

```diff
@@ -88,6 +84,23 @@ def check(condition, msg):
         )
         check("[pypi]\n" in pypirc_text, "[pypi] section not found in ~/.pypirc")
 
+    output = subprocess.run(
+        ["check-manifest", root_dir],
+        stdout=subprocess.PIPE,
+        stderr=subprocess.STDOUT,
+        encoding="utf-8",
+    )
+    check(output.returncode == 0, f"Error in manifest:\n{output.stdout}")
+
+    output = subprocess.run(
+        f"python setup.py checkdocs".split(),
```

Comment:
In the deploy script I switched to using `twine check`, which I think should do the same thing, but I don't know to what extent that works before making the actual releaes. Just trying to keep deps down :shrug:

Reply:
Ah yeah that's even better then :+1: